firmware/library/L3_HAL/apds_9960.hpp

```diff
@@ -166,26 +167,24 @@ class Apds9960 : public Apds9960Interface
     kAiht                   = 0x00
   };
 
+  inline static I2c i2c;
```

Comment:
Add a TODO() to move this out of static, so that this HAL can be platform agnostic.